web/test/pages/roadmap.test.tsx

```diff
@@ -228,4 +239,132 @@ describe("roadmap page", () => {
     expect(within(sectionStart).getByText("step2")).toBeInTheDocument();
     expect(within(sectionStart).getByText("step4")).toBeInTheDocument();
   });
+
+  it("displays sections based on default state", () => {
```

Comment:
small nitpick, but technically this test isn't testing a "default" state. it displays sections based on whatever are the open sections in the userData preferences. the fact that they happen to be "default" is really just something we know. But in the eyes of this test, there's nothing default about it. It's just displaying whatever sections are in the preferences array

Reply:
Addressed.